compiler/optimizer/Inliner.cpp

```diff
@@ -103,6 +103,7 @@
 #include "ras/DebugCounter.hpp"
 #include "ras/LogTracer.hpp"                              // for heuristicTrace, etc
 #include "runtime/Runtime.hpp"
+#include "compile/SymbolReferenceTable.hpp"               // for osrFearPointSymbol, etc
```

Comment:
Can we keep the alphabetical order and move this up to the other `"compile/"` includes?

Reply:
looks like it is already included, i deleted this line.